src/PerforceService.ts

```diff
@@ -166,6 +166,14 @@ export namespace PerforceService {
         } catch {}
         return false;
     }
+	
+	async function isSymbolicLinkAndDirectory(uri: Uri): Promise<boolean> {
+        try {
+            const linkAndDir = FileType.SymbolicLink | FileType.Directory;
+            return (await workspace.fs.stat(uri)).type === linkAndDir;
```

Comment:
It's not a big deal but since fs.stat is an async filesystem operation I think it would be better to just call it once to check for both directory or dir + symlink

Reply:
Thanks, Matt.  I'll work with Priya to see if we can implement (and test) this suggestion quickly. 

Cheers,
--Phil

ps. Big thank you on this.  It will really make a huge difference to our development.

Reply:
@mjcrouch Thank you for taking time to review this change. I've added the symlink +directory evaluation in the existing function 'isDirectory'.